.github/workflows/build.yml

```diff
@@ -4,6 +4,10 @@ on:
   push:
   pull_request:
 
+permissions:
+  contents: read
+  id-token: write
```

Comment:
What is this for?

Reply:
See the PR description. I created it from the CLI and was writing the description when you commented. You're too fast 😅